app/services/teacher_training_public_api/sync_all_providers_and_courses.rb

```diff
@@ -0,0 +1,30 @@
+module TeacherTrainingPublicAPI
+  class SyncAllProvidersAndCourses
+    def self.call
+      begin
+        (1..).each do |page|
+          sync_providers(
+            TeacherTrainingPublicAPI::Provider.where(year: 2021).paginate(page: page, per_page: 500).all,
```

Comment:
I saw this here (and in Course below) and raised one eyebrow, then both, then actually read your PR description...